README.markdown

```diff
@@ -279,23 +275,23 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
+* Colons should have one space on the left and one on the right. Exceptions are empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
```

Comment:
Hum, maybe I see this in a wrong context, but if this is generalised does this mean that the notation like ```var data : [String : CGFloat]``` as well as ```func foo(_ param : String)``` are preferred?